pkg/prowgen/prowgen_config.go

```diff
@@ -19,12 +19,15 @@ type Repository struct {
 	Org                   string                                                      `json:"org,omitempty" yaml:"org,omitempty"`
 	Repo                  string                                                      `json:"repo,omitempty" yaml:"repo,omitempty"`
 	Promotion             Promotion                                                   `json:"promotion,omitempty" yaml:"promotion,omitempty"`
+	BinaryBuildCommands   string                                                      `json:"binaryBuildCommands,omitempty" yaml:"binaryBuildCommands,omitempty"`
 	ImagePrefix           string                                                      `json:"imagePrefix,omitempty" yaml:"imagePrefix,omitempty"`
 	ImageNameOverrides    map[string]string                                           `json:"imageNameOverrides,omitempty" yaml:"imageNameOverrides,omitempty"`
 	SlackChannel          string                                                      `json:"slackChannel,omitempty" yaml:"slackChannel,omitempty"`
 	CanonicalGoRepository *string                                                     `json:"canonicalGoRepository,omitempty" yaml:"canonicalGoRepository,omitempty"`
 	E2ETests              []E2ETest                                                   `json:"e2e,omitempty" yaml:"e2e,omitempty"`
 	Dockerfiles           Dockerfiles                                                 `json:"dockerfiles,omitempty" yaml:"dockerfiles,omitempty"`
+	BaseImages            map[string]cioperatorapi.ImageStreamTagReference            `json:"baseImages,omitempty" yaml:"baseImages,omitempty"`
+	SharedInputs          map[string]cioperatorapi.ImageBuildInputs                   `json:"sharedInputs,omitempty" yaml:"sharedInputs,omitempty"`
```

Comment:
Wouldn't it make sense to keep variable name as per CI operator API, `ImageBuildInputs`?

Reply:
Yes, I can name it `ImageBuildInputs` although the shared aspect differs here from CI operator API.

In CI operator API, the `ImageBuildInputs` is used for a specific image. Here, I'm using that do add common, provided upfront inputs for all images from a given repo.

I could name it `SharedImageBuildInputs`, but isn't that too mouthful?